mobile/android/build.gradle

```diff
@@ -1,5 +1,6 @@
 ext {
-    appCompatVersion = '1.1.0' // for background_fetch
```

Comment:
Will this change break `background_fetch`?

Reply:
This was made to fix the compilation AFAIK, so it is compiling without it as well. we can test if it was made otherwise.

Reply:
I have separated out this code in new PR, let's have discussion there.

Reply:
Will let @ua741 take over 🙏